### DISCUSSION TITLE
Open a settings page from camera apps interface (acap3 like index.html)

### DISCUSSION THREAD

**Discussion by ahosnyyy** [AxisCommunications/acap-computer-vision-sdk-examples#116] [General] [answered]
In acap3 manifest.json file, I was able to set the settings page like:
"configuration": {
            "settingPage": "index.html"
}

is there an alternative in the new cv sdk?

**Comment by Corallo** [comment, marked as answer]
Hello @ahosnyyy 

The short answer, is that at the moment there is no way to do something equivalent to the acap3 setting page in the CV sdk.
We are aware of the limitation, but for now we don't have plans to extend the support for it.


However, if you need something similar, you could implement your own server inside the container application, and allow connection from outside mapping some ports in the docker compose file. (Something similar to our [web-server-example](https://github.com/AxisCommunications/acap-computer-vision-sdk-examples/tree/main/web-server)). What is not possible is to have a button in the camera GUI that bring you to the setting page.

Hope this is helpful
